doc/cla/corporate/maao.md

```diff
@@ -1,12 +1,12 @@
 Kyiv, Ukraine, 2017-05-29
 
-Management and Accouting online llc agrees to the terms of the Odoo Corporate Contributor License Agreement v1.0.
+Management and Accounting online llc agrees to the terms of the Odoo Corporate Contributor License Agreement v1.0.
 
 I declare that I am authorized and able to make this agreement and sign this declaration.
 
 Signed,
 
-Management and Accouting online llc info@maao.com.ua https://github.com/management-and-acounting-on-line
+Management and Accounting online llc info@maao.com.ua https://github.com/management-and-acounting-on-line
```

Comment:
I wouldn't modify the CLA of a contributor, regardless of any typos

Reply:
Reverted in 39aa2d2